go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/otiai10/copy v1.14.1
 	github.com/stretchr/testify v1.10.0
 	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415
+	github.com/yuin/goldmark v1.7.12
```

Comment:
There is a new patch version v1.7.13 published a few days ago
https://github.com/yuin/goldmark/releases/tag/v1.7.13

Could you try to update it here?